tests/testthat/helper-demo.R

```diff
@@ -4,14 +4,13 @@ setup_tasks_demo <- function() {
   dirinfo <- list(
     wd = getwd(),
     #srcdir = 'inst/extdata/tasks_demo', # for use while developing tests
-    srcdir = system.file('extdata/tasks_demo', package='scipiper'),
+    srcdir = system.file('inst/extdata/tasks_demo', package='scipiper'),
```

Comment:
This was the change that allowed system.file to work equally well in console and `devtools::test()` environments